spec/types/calculation.md

```diff
@@ -252,12 +252,22 @@ This algorithm takes a calculation `calc` and returns a number or a calculation.
 * If `calc`'s name is `"calc"` and `arguments` contains exactly a single number
   or calculation, return it.
 
-* If `calc`'s name is `"mod"`, `"rem"`, `"atan2"`, or `"pow"`; `arguments` has
-  fewer than two elements; and none of those are unquoted strings, throw an
-  error.
+* If `calc`'s name is `"calc"`, `"sin"`, `"cos"`, `"tan"`, `"asin"`, `"acos"`,
+  `"atan`", `"sqrt"`, `"log"`, `"exp"`, `"sign"` and `arguments` has more than
+  one element, throw an error.
+
+* If `calc`'s name is `"mod"`, `"rem"`, `"atan2"`, `"pow"`, or `"calc-size"`:
+
+  * If `arguments` has fewer than two elements; and none of those are unquoted
+    strings, throw an error.
+
+    > It's valid to write `pow(var(--two-args))` or `pow(#{"2, 3"})`, but
+    > otherwise calculations' arguments must match the expected number.
```

Comment:
Why is this only valid for calculations with two arguments? We could also have `round(var(--three-args))` or `round(#{"2, 3, 4"})` right?

Reply:
`round()` and `clamp()` handle their fewer-than-three-arg cases below.